qa/suites/rbd/ro/clusters/fix-3.yaml

```diff
@@ -0,0 +1,8 @@
+roles:
+- [mon.a, mon.c, mgr.x, osd.0, osd.1, osd.2, osd.3]
```

Comment:
Nit: you can probably drop this down to a two node test and reduce the number of MONs and OSDs. The fewer the number of required nodes, the easier it is to schedule the test.